i18n/hu.i18n.json

```diff
@@ -811,5 +811,15 @@
     "Addig a __communityName__ közösségben az ön hozzászólásai és szavazatai mellett a többi tulajdonos hivatalos névként azt fogja látni: __personName__."],
 
   "s": "k",
-  "":""
+  "":"",
+
+  // Months (in charts) 
+  "FEB": "FEBR",
```

Comment:
Erdemes ilyenkor a JAN -> JAN -t is beleirni, csak azert mert ha valaki szuahelira akarja majd forditani a programot, akkot azt fogja csinalni, hogy veszi ezt a forditasi fajlt, es abban atirja a forditasokat.